CMake/itkFFTImageFilterFactoryRegisterManager.h.in

```diff
@@ -0,0 +1,56 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef itkFFTImageFilterFactoryRegisterManager_h
+#define itkFFTImageFilterFactoryRegisterManager_h
+
+namespace itk {
+
+class FFTImageFilterFactoryRegisterManager
+{
+  public:
+  explicit FFTImageFilterFactoryRegisterManager(void (* const list[])(void))
+    {
+    for(;*list != nullptr; ++list)
+      {
+      (*list)();
+      }
+    }
+};
+
+
+//
+//  The following code is intended to be expanded at the end of the
+//  itkTransformFileReader.h and itkTransformFileWriter.h files.
+//
+@LIST_OF_FACTORIES_REGISTRATION@
+
+//
+// The code below registers available IO helpers using static initialization in
+// application translation units. Note that this code will be expanded in the
+// ITK-based applications and not in ITK itself.
+//
+
+void (* const FFTImageFilterFactoryRegisterRegisterList[])(void) = {
+  @LIST_OF_FACTORY_NAMES@
+  nullptr};
```

Comment:
Is this global `RegisterRegisterList` variable, actually used anywhere outside itkFFTImageFilterFactoryRegisterManager.h.in itself? If it is only just an internal implementation detail, it would be better make it a local variable. (Possibly local inside the constructor of the manager.)

Reply:
Thanks @tbirdso! Honestly I'm considering to propose removing the global RegisterRegisterList variables from the IOFactoryRegisterManager.h.in files as well (as they don't seem to be used anyway), but only after the release on v5.3 😃

Reply:
Agreed, `RegisterRegisterList`s don't seem to (and probably shouldn't) be used anywhere else beside these files. Good thinking @N-Dekker .